### PR TITLE
Fix total amount mismatch when select multiple payment method

### PR DESCRIPTION
**Pull request by andykim**
## Description
- Fix total amount mismatch when select multiple payment method
https://xendit.atlassian.net/browse/TPI-7941

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] General improvement (non-breaking feature that might not necessarily impact customer)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new **vulnerabilities**
- [ ] I have added tests that prove my fix is effective or that my feature works

## Merging
- [ ] I am aware which environment I intend to merge (staging | production)
- [ ] I am aware that a deployment will be triggered if I merge to staging and release branch
- [ ] I informed the team about my merge if it has risk of breaking the target API

## Deployment
- [ ] Unit tests passed
- [ ] UAT passed and communicated to QA and Product is informed about this deployment
- [ ] I'm aware off all the commits / merges behind that will be included in this deployment
- [ ] Feature flags are ready and communicated with the team

Please follow our [deployment guideline](https://xendit.atlassian.net/wiki/spaces/TPI/pages/2331476248/Engineer+Onboarding#Releasing-Flow).

## Screenshot

It will show the error message in a popup if the total amount <= 0

Screenshot from 2022-08-24 14-55-20

### CODE REVIEW

File: xendit_pos/static/src/js/payment_xendit_pos.js [hunk 1]
```diff
@@ -65,7 +66,19 @@ odoo.define('xendit_pos.payment', function (require) {
             const self = this;
 
             const order = this.pos.get_order();
+            const payment_line = order.selected_paymentline;
+
+            if(payment_line.amount <= 0){
+                // TODO check if it's possible or not
```
Comment (ikanyu) [inline xendit_pos/static/src/js/payment_xendit_pos.js:72]: I think this might need to be reworded to explain what are we checking what's possible

Comment (andykim) [reply xendit_pos/static/src/js/payment_xendit_pos.js]: Updated @ikanyu

File: xendit_pos/static/src/js/payment_xendit_pos.js [hunk 2]
```diff
@@ -65,7 +66,17 @@ odoo.define('xendit_pos.payment', function (require) {
             const self = this;
 
             const order = this.pos.get_order();
+            const payment_line = order.selected_paymentline;
+            if(payment_line.amount <= 0){
```
Comment (ikanyu) [inline xendit_pos/static/src/js/payment_xendit_pos.js:70]: one last comment 😬 need empty spaces beside the `()`

**Review by ikanyu — APPROVED** [review]
One comment else LGTM

**Review by yanoandri — APPROVED** [review]

**Pull request merged**